src/libs/actions/Report.js

```diff
@@ -198,6 +204,33 @@ function setLocalLastRead(reportID, sequenceNumber) {
     });
 }
 
+/**
+ * Remove a specific action from a report by index.
+ *
+ * @param {Number} reportID
+ * @param {String} actionID
+ */
+function removeLocalAction(reportID, actionID) {
```

Comment:
Does it matter if it's a local action or not? I don't think so. I would suggest just renaming this to `removeAction` since it can be used to remove any action ID.

Reply:
I think mainly my reasoning was that we are in the actions file and it is less specific to say `removeAction` 
e.g. that could be confused with an API call that deletes an action. Not sure if that is important at this stage.

Reply:
`addAction()` calls the API `removeAction()` does not

Reply:
Maybe it would be clearer if this was `removeActionFromStorage()`

Reply:
actually I think we can just axe this and bulk remove everything